modules/common/variables.tf

```diff
@@ -12,13 +12,24 @@ variable "environment_name" {
   }
 }
 
+variable "instance_disk_size" {
+  description = "Size of the disk attached to the cluster instance."
+  default     = 50
+  type        = number
+}
+
 variable "instance_types" {
   description = "Instance types that is preferred for node group."
   type        = list(string)
 }
 
-variable "desired_capacity" {
-  description = "Desired number of nodes that the node group should launch with initially."
+variable "min_cluster_capacity" {
+  description = "Minimum number of EC2 nodes."
```

Comment:
similar to my comment above on `nodes` vs `instances` - https://github.com/atlassian-labs/data-center-terraform/pull/188/files#r830717859